newsletters/2023-09.md

```diff
@@ -0,0 +1,63 @@
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting soon; provide a summary of this year's BOSC in Lyonl; share news from the OBF event fellowship; and announce that we are looking for someone to support us in being the organisation-admin for the 2024 Google Summer of Code. 
+
+In longer form: 
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having it's next public board meeting at XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog. 
+
+### BOSC news
+
+This year we had the 24th annual Bioinformatics Open Source Conference, which took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceeded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
+
+In addition to XXX talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
+
+You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).
+
+### OBF Travel Fellowships
+
+Next round: December 1, 2023
+Last round: August 1, 2023
+
+The ongoing application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
+
+These fellowship can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
+
+### Google Summer of Code
+
+We are looking for a motivated person who would like to volunteer as the organisation admin to allow the OBF to participate in the 2024 Google Summer of Code. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 
```

Comment:
```suggestion
We are looking for a motivated person who would like to volunteer as the organisation admin to enable the OBF to participate in the 2024 Google Summer of Code, as we have in previous summers. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 
```